mne_icalabel/megnet/label_components.py

```diff
@@ -0,0 +1,107 @@
+from importlib.resources import files
+
+import numpy as np
+import onnxruntime as ort
+from mne.io import BaseRaw
+from mne.preprocessing import ICA
+from numpy.typing import NDArray
+
+from .features import get_megnet_features
+
+_MODEL_PATH: str = files("mne_icalabel.megnet") / "assets" / "megnet.onnx"
```

Comment:
I think this line causes issues. 

See: https://github.com/mne-tools/mne-icalabel/blob/43eec6dbce9cf9267e1f2de96d0adc3835c92268/mne_icalabel/iclabel/network/onnx.py#L27

Let us know if that doesn't work.

Reply:
I don't think he modified anything from `iclabel`, that should not be failing.

Reply:
Yes nvm. I misread the error. 

Hmm it seems the file is somehow not found.

Reply:
Oh, I found the issue! After I added the `__init__.py `file in the `mne_icalabel/megnet` directory, it was able to correctly find `megnet.onnx`. (but I remember that python(at least v3.8)  don't require __init__.py to treat a folder as a package.)